src/Telemetry/Nuget.config

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
we should just use the nuget.config in the source of the project (it will be automatically used if this is missing).

Reply:
Sure. Removed.